src/core/chain.ts

```diff
@@ -294,7 +294,7 @@
             chainsByLayer[layer].sort((a: IChain, b: IChain) => { return b.pathLength - a.pathLength });
 
             if (opts.contain) {
-                var containedChains = getContainment(chainsByLayer[layer], opts.contain);
+                var containedChains = getContainment(chainsByLayer[layer], {alernateWindings:true});
```

Comment:
no merit, the compiler did the work :-)

I think that might work. alernatewinding is indeed a typo in the code, we keep it for backward compatibility ?